playbooks/tests/tasks/integration.yml

```diff
@@ -125,3 +112,74 @@
            cirros@{{ floating_ip.floating_ip.floating_ip_address }}
            ping -c 5 www.google.com
   when: floating_ip.floating_ip.floating_ip_address is defined
+
+- name: nova can attach cinder volume
+  environment:
+    PYTHONPATH: "{{ basevenv_lib_dir|default(omit) }}"
+  os_server_volume:
+    auth:
+      auth_url: "{{ endpoints.auth_uri }}"
+      project_name: admin
+      username: admin
+      password: "{{ secrets.admin_password }}"
+    server: turtle-stack
+    volume: turtle-vol
+  when: cinder.enabled|bool
+
+- name: write volume in VM
+  shell: ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no
+         -i /root/.ssh/id_rsa
+         cirros@{{ floating_ip.floating_ip.floating_ip_address }}
+         sudo dd if=/dev/zero of=/dev/vdb count=100 bs=1M
+  when: cinder.enabled|bool
+
+- name: detach volume to instance
+  environment:
+    PYTHONPATH: "{{ basevenv_lib_dir|default(omit) }}"
+  os_server_volume:
+    auth:
+      auth_url: "{{ endpoints.auth_uri }}"
+      project_name: admin
+      username: admin
+      password: "{{ secrets.admin_password }}"
+    server: turtle-stack
+    volume: turtle-vol
+    state: absent
+  when: cinder.enabled|bool
+
+- name: delete volume created via cinder
+  environment:
+    PYTHONPATH: "{{ basevenv_lib_dir|default(omit) }}"
+  os_volume: 
```

Comment:
seems the [os_volume](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/cloud/openstack/os_volume.py#L133) module doesn't check if the volume is really disappeared after sending `delete` request.
So I think we can use shell to delete volume, shell will be simpler, with shell we can even more do many things in a loop
```
- name: delete resources
  shell: source /root/stackrc && {{ item }}
  with_items:
     - openstack server remove volume turtle-stack turtle-vol
     - nova delete turtle-stack
     - cinder delete turtle-vol
     ......
```